src/active_query.rs

```diff
@@ -60,6 +60,9 @@ pub(crate) struct ActiveQuery {
 
     /// Provisional cycle results that this query depends on.
     cycle_heads: CycleHeads,
+
+    /// If this query is a cycle head, iteration count of that cycle.
+    pub(crate) iteration_count: u32,
```

Comment:
Should we make this private and provide an accessor instead? It would make it clear that no-one is writing the field outside of `ActiveQuery`.